great_expectations/rule_based_profiler/expectation_configuration_builder/default_expectation_configuration_builder.py

```diff
@@ -37,30 +77,141 @@ def __init__(
         if not isinstance(meta, dict):
             raise ge_exceptions.ProfilerExecutionError(
                 message=f"""Argument "{meta}" in "{self.__class__.__name__}" must be of type "dictionary" \
-(value of type "{str(type())}" was encountered).
+(value of type "{str(type(meta))}" was encountered).
 """
             )
 
+        if condition and (not isinstance(condition, str)):
+            raise ge_exceptions.ProfilerExecutionError(
+                message=f"""Argument "{condition}" in "{self.__class__.__name__}" must be of type "string" \
+(value of type "{str(type(condition))}" was encountered).
+"""
+            )
+
+        self._condition = condition
         self._meta = meta
 
     @property
     def expectation_type(self) -> str:
         return self._expectation_type
 
     @property
-    def kwargs(self) -> dict:
-        return self._kwargs
+    def condition(self) -> str:
+        return self._condition
 
     @property
     def meta(self) -> dict:
         return self._meta
 
+    @property
```

Comment:
@NathanFarmer The property `kwargs` goes before property `meta` -- minor point, but helps readability.  Thanks!

Reply:
✅